src/agent/sqlite_manager/src/sqlite_manager.cpp

```diff
@@ -164,12 +166,13 @@ namespace sqlite_manager
         catch (const std::exception& e)
         {
             std::cerr << "Error during Retrieve operation: " << e.what() << '\n';
-            return {};
+            throw;
         }
         return results;
```

Comment:
Looking at this, I think we can remove the return from line [164](https://github.com/wazuh/wazuh-agent/pull/112/files#diff-361d375833f9847751df2dd9f10feadf9423fb3f5d3e87386905d1911d28ea1aR164).

Reply:
Done.